infrastructure/kong/README.md

```diff
@@ -1,50 +1,31 @@
 # KONG API Service Portal Setup
 
 The public API is accessible at https://bcparks.api.gov.bc.ca (PROD) or
-https://bcparks-api-gov-bc-ca.test.api.gov.bc.ca (TEST). Dev does not have a corresponding API Gateway.
+https://bcparks.test.api.gov.bc.ca (TEST). Dev does not have a corresponding API Gateway.
```

Comment:
I think this sentence is incorrect.  https://bcparks.test.api.gov.bc.ca isn't working currently and you have to use https://bcparks-api-gov-bc-ca.test.api.gov.bc.ca

Reply:
Good catch!